gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -995,35 +1054,57 @@ gst_tensor_converter_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
       g_assert ((buf_size % frame_size) == 0);
       frames_in = buf_size / frame_size;
       break;
-    case _NNS_MEDIA_PLUGINS:
-    {
-      GstTensorsConfig new_config;
-
-      if (self->externalConverter == NULL ||
-          self->externalConverter->convert == NULL)
-        return GST_FLOW_NOT_SUPPORTED;
-
-      inbuf =
-          self->externalConverter->convert (buf, &frame_size, &frames_in,
-          &new_config);
-
-      if (gst_tensors_config_is_equal (config, &new_config) != TRUE) {
-        if (gst_tensor_converter_update_caps (self, self->srcpad,
-                &new_config) == TRUE) {
-          self->tensors_config = new_config;
-        } else {
+    case _NNS_MEDIA_ANY:
+      if (self->mode == _CONVERTER_MODE_CUSTOM) {
+        GstTensorsConfig new_config;
+        if (self->custom.func == NULL) {
+          nns_loge
+              ("custom condition of the tensor_converter is not configured.");
           return GST_FLOW_ERROR;
         }
-      }
-
-      g_assert (inbuf != NULL);
-      g_assert (frame_size > 0);
+        inbuf = self->custom.func (buf, &new_config);
```

Comment:
plz add condition if failed to convert buffer (inbuf is null)

Reply:
Updated.